package.json

```diff
@@ -69,5 +69,5 @@
     "uglify-js": "^2.4.14",
     "wd": "^0.3.8"
   },
-  "version": "2.3.1"
+  "version": "2.3.2"
```

Comment:
I don't think we should bump the version inside PR. I also don't think this is a patch level change.